content/docs/zero/upgrading.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Upgrading
+description: This page discusses how to upgrade your cluster to use the latest version of Pomerium Zero.
+keywords:
+  [
+    pomerium zero release,
+    zero release,
+    zero releases,
+    pomerium zero releases,
+    upgrade zero,
+    zero version,
+    latest zero version,
+  ]
+pagination_prev: null
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade Pomerium Zero
```

Comment:
I think we should be careful with our terminology. What does "Pomerium Zero" refer to? I'd expect this to mean "our managed control plane", in which case I'm not sure it makes sense to talk about "upgrading Pomerium Zero."

How do you think we should talk about this?

I'd suggest talking about "upgrading Pomerium" or "upgrading your Pomerium Zero deployment."